README.md

```diff
@@ -1,17 +1,31 @@
 # yesod-recaptcha2
 
 ~~~hs
-import Import.ReCaptcha2
+import Yesod.ReCaptcha2
 ~~~
 
 ~~~hs
 instance YesodReCaptcha App where
-    reCaptchaSiteKey = return "foo"
-    reCaptchaSecretKey = return "bar"
+    reCaptchaSiteKey = pure "foo"
+    reCaptchaSecretKey = pure "bar"
+    reCaptchaLanguage = pure Nothing
+
+    -- with specific language from https://developers.google.com/recaptcha/docs/language
+    -- reCaptchaLanguage = pure (Just "ru")
 ~~~
 
 
 ~~~hs
 <* reCaptcha
 ~~~
 
+Append to applicative form:
+
+~~~hs
+buildForm :: Form MyForm
+buildForm = renderDivs $ MyForm
+  <$> areq textField myFieldSettings Nothing
+  <* reCaptcha
+  where
+    myFieldSettings = ...
+~~~
```

Comment:
I think that one sample code is enough

Also, I think that it is better to make it compileable

Reply:
Ok :)